### ISSUE TITLE
New update for the season two. From Hamster @shamhi

### ISSUE THREAD

**Issue by RedWilly** [shamhi/HamsterKombatBot#2447]
🛡 PROTECTING THE HAMSTER 🛡

👿 We’ve banned over 2.3 million cheaters — enough to fill the entire city of Rome!

🏴‍☠️ Wondering how we caught them? Well, here’s an example: One person connected over 400 accounts to a single Binance address, while another invited almost 2,000 "friends," all of whom were flagged by our anti-cheat system.

🤖 We have been analyzing users’ behavior for months before the TGE. We have seen accounts that logged in at the same time each day and were tapping the same exact pixel. 

@shamhi

**Comment by RedWilly** [comment]
> This script doesn't have these problems.

Yeah but the tap does. It instant tap all at once and in the same spot

**Comment by Ankor12** [comment]
> This script doesn't have these problems.

I get banned because of this script

**Comment by RedWilly** [comment]
> > This script doesn't have these problems.
> 
> I get banned because of this script

to be frank no one told you to use this script- you did so in your own free will. 

the only solution now it 
1. dont use this in the next season or 
2.  use it anyways and hope they dont flag your accounts this time

**Comment by Ankor12** [comment]
> > > This script doesn't have these problems.
> > 
> > 
> > I get banned because of this script
> 
> to be frank no one told you to use this script- you did so in your own free will.
> 
> the only solution now it
> 
> 1. dont use this in the next season or
> 2. use it anyways and hope they dont flag your accounts this time

Can you tell me when I complained?
I couldn't care less about this hamster, I tried the script, it didn't work.

I was just replying to an erroneous message which said that the script had no problem with the banns.

**Comment by euphoria360** [comment]
> > > > This script doesn't have these problems.
> > > 
> > > 
> > > I get banned because of this script
> > 
> > 
> > to be frank no one told you to use this script- you did so in your own free will.
> > the only solution now is
> > 
> > 1. dont use this in the next season or
> > 2. use it anyways and hope they dont flag your accounts this time
> 
> Can you tell me when I complained? I couldn't care less about this hamster, I tried the script, it didn't work.
> 
> I was just replying to an erroneous message which said that the script had no problem with the banns.

Its on you, not Shamhi or other devs.

I used this script for 4 months. It got me more than 8 millions profit per hour. didnt get banned. you needed to tweak the options. its not a copy/paste job!

**Comment by Praveensenpai** [comment]
I got banned.

I didn't used this script or key generator. still my account got cheat option.

tbh I think they randomly pick and give them cheap card.

**Comment by RedWilly** [comment]
Did you used any other scripts. Because I know the most of us that got banned were using bots

**Comment by Praveensenpai** [comment]
Man, this is so frustrating! I didn’t use any scripts on my main account—I just played games every day for two weeks and got bored because they kept adding more. 

But on another account, which was on a different device, I tried a script and hosted it on Railway. Now both accounts got flagged for cheating. I seriously regret not using that script on my main account because they banned both, no matter if it was a bot or actual gameplay. What a mess!

**Comment by euphoria360** [comment]
What i did when using this script was that:
- i changed every default variable to something custom. I also checked them every few weeks and tweaked them based on my profit.
- i used a private ip on my vps for the script.
- and from time to time, maybe once a week or 2 weeks i looked at my account on my telegram to see if things go smoothly.
- i checked for new bot versions from time to time.
- i stopped the script few hours before snapshot.

Thats all i did. Hope it helps others too
